src/main/java/com/massivecraft/factions/zcore/util/TL.java

```diff
@@ -625,7 +625,7 @@
     PLAYER_USE_WILDERNESS("<b>You can't use <h>%s<b> in the wilderness."),
     PLAYER_USE_SAFEZONE("<b>You can't use <h>%s<b> in a safe zone."),
     PLAYER_USE_WARZONE("<b>You can't use <h>%s<b> in a war zone."),
-    PLAYER_USE_TERRITORY("<b>You can't use <h>%s<b> in the territory of <h>%s<b>."),
+    PLAYER_USE_TERRITORY("<b>You can't <h>%s %s<b> in the territory of <h>%s<b>."),
```

Comment:
You're introducing an extra string here. Why replace use when this is only called when they are using something?